pallets/msa/src/types.rs

```diff
@@ -48,6 +50,12 @@ pub struct AddDelegate {
 	pub permission: u8,
 }
 
+#[derive(TypeInfo, Debug, Clone, Decode, Encode, PartialEq)]
+pub struct AccountCreationDelegate<AccountId> {
+	pub key: AccountId,
```

Comment:
i think this the msa_id?

Reply:
should this be the msa_id that you are giving permission to sponsor your account creation?

Reply:
you want to sign who you are giving permission to, what msa.

Reply:
you can sign whatever the payload is as long as the payload and signature are valid, but for the sake of consistency I'll remove this and use the existing one